statsmodels/genmod/families/family.py

```diff
@@ -736,6 +739,8 @@ class Binomial(Family):
     links = [L.logit, L.probit, L.cauchy, L.log, L.cloglog, L.identity]
     variance = V.binary  # this is not used below in an effort to include n
 
+    safe_links = [L.logit, L.CDFLink]
```

Comment:
ok, I checked: cloglog is a subclass of Logit so should be covered.
maybe a comment in the code as a reminder: something like `# other safe links are subclasses`